File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.java
class Solution {
    public int solution(String binomial) {
        int answer = 0;
        
        String[] str=binomial.split(" ");
		
		int arr=Integer.parseInt(str[0]);
		int arr1=Integer.parseInt(str[2]);
		
		if(str[1].equals("+")) {
			answer=arr+arr1;
		}
		else if(str[1].equals("-")) {
			answer=arr-arr1;
		}
		else if(str[1].equals("*")) {
			answer=arr*arr1;
		}
        
        
        return answer;
    }
}

File: 프로그래머스/0/181865. 간단한 식 계산하기/간단한 식 계산하기.js
function solution(binomial) {
    var answer;
    
    const[a, op, b]  = binomial.split(' ');
    let Num1=Number(a);
    let Num2=Number(b);
    
    if(op=='+'){
        answer=Num1+Num2;
    }else if(op=='-'){
        answer=Num1-Num2;
    }else if(op=='*'){
        answer=Num1*Num2;
    }
    
    
    
    
    return answer;
}